### ISSUE TITLE
Section header graphic distinction

### ISSUE THREAD

**Issue by kuzunguka** [mimbrown/render#112]
For discussion/feedback from local reviewers: I really like the new 1st level section header.  However, the 2nd and 2rd level headers are a bit confusing to me.  Within the idiomatic text, I would still love to see the 3 types of section headers distinguished by the size of the type or size of the graphic.  Currently the type and graphic are all the same size.  In places where a 2nd level section header is immediately followed by a 3rd level section header (e.g. Matt 5:21) it just looks weird and confusing. Likewise the difference between level 1 and level 2 (e.g. at the beginning of Matt 5 where they’re both visible) is confusing - my first reaction (not knowing that they are supposed to be two different types of headers) is that someone forgot to complete the bottom side of the fancy graphic.  Idea 1: different type sizes.  Idea 2: use graphics similar to what the printed book has - these graphics are clearly three different levels of fanciness/size. Idea 3: right justify level 2 and 3 headers, or just level 3 headers (probably necessitating changing or eliminating the graphic)

**Comment by mnjames** [comment]
My own opinion is that most people won't understand the 3-level header distinction no matter how we picture it. I assume most people don't notice the distinction in the book. I'd recommend relying on the graphic designer, Cathy's, suggestion unless we have clear direction that a different method is different. I personally don't like the idea of smaller fonts.

Personally I'd definitely prefer we simply got rid of all cases of two levels of headers together, and probably also all distinction between different levels.

**Comment by socvid** [comment]
I agree with @mnjames that I would be happy for level 2 and 3 headers not to exist.  There are so few in the whole book that it makes me wonder why bother having them at all?  They should either be promoted to level 1 or removed.  My opinion is to promote them because for example Matt 5's header levels 2 and 3 all exist in my English translations as level 1 headers.  In fact I'm not familiar with an English translation that uses different levels of headers, although I wouldn't doubt that one exists.  It also would be annoying to me to remove the level 2 and 3 headers all together and cause a chapter to not have any headers, but this is probably me being too rigid in my thinking.

On the other hand I agree with @kuzunguka in that I do think it is possible to communicate different levels of headers, and I don't think the level 2 and 3 headers did this as well as they could.  I think font size could be helpful, but then again header's font size might get out of control large.  So instead of changing font size my suggestion would be to keep the beautiful level 1 header, use level 3's simple box graphic as level 2, and make level 3 headers not have a box but a graphic above and below the text like it is in the book and how it was originally in the first release of this app.  The graphic could be what was originally used, or inspired by the level 1 header, or something else.  I believe level 1 headers are always followed by verses before a level 2 header is used, but level 3 headers can immediately follow a level 2 header.  If this is the case, then I think it will look better not having two boxed headers right next to each other.

My preference would probably still be to promote all level 2 and 3 headers to level 1, but I think I would also like seeing headers redone in the way I described.

**Comment by socvid** [comment]
Faisal hadn't read a chapter with different headers.  I directed him to Matthew 5, and he said that he understood the distinctions and was fine with how they were.

**Comment by kuzunguka** [comment]
I agree with @socvid and @mnjames: My own preference is also that we promote all level 2 and 3 headers to level 1 headers.  Or at least promote level 3 headers to level 2 headers.  I believe that it would be easier for our audience to navigate with only 1, or if necessary 2, levels of headers rather than 3.  However, if we must retain 2 or 3 levels of headers, I would campaign for seeing if Cathy can change the graphic somehow.  I think the whole point of having different levels of headers is lost unless we display them more clearly with differently-sized graphics or differently-sized header type.

**Comment by kuzunguka** [comment]
[We were specifically look through Matt 5-7; I did not explain how often this does or does not occur in the whole app.]

AH: 
3rd-level headers should simply be removed (not converted to 2nd- or 1st-level), especially when a 3rd-level header immediately follows a 2nd-level header.  Otherwise readers will be confused. Two headers next to each other is not good. But also when a 3rd-level header is by itself it can be removed.

[JD: of course this would lead to the problem of a few chapters having no headers at all.]
